recipes/renode-cli/variants.yaml

```diff
@@ -0,0 +1,7 @@
+c_stdlib:
+    - if: linux
+      then: sysroot
+    - if: osx
+      then: macosx_deployment_target
+    - if: not unix
+      then: vs
```

Comment:
```suggestion
```
You shouldn't need this file in `conda-forge` - these defaults are already supposed to set in both staged-recipes and the feedstock.  You might need it to build locally, but it shouldn't be included in the recipe submission.